public/index.html

```diff
@@ -172,26 +179,16 @@ <h1 class="text-primary"><abbr
             loadOutputEditor();
             setValueOutputEditor(getDefaultText());
             generateDefaultTable();
-
-            console.log("Adding scroll listener");
-
-            $("#tableButton").click(function () {
```

Comment:
Removed Global Click Handler as it is not really needed. Need to scroll only if JSON is valid.